test/unittest/layers/layers_dependent_common_tests.cpp

```diff
@@ -124,12 +124,13 @@ TEST_P(LayerSemantics, setBatchValidateLayerNode_p) {
 
 #ifdef ENABLE_OPENCL
 TEST_P(LayerSemanticsGpu, createFromClContext_pn) {
-  auto &ac = nntrainer::ClContext::Global();
+  auto &eg = nntrainer::Engine::Global();
+  auto ac = static_cast<nntrainer::ClContext *>(eg.getRegisteredContext("gpu"));
```

Comment:
`ac` sounds like `appContext`. What about naming it `cc` instead of `ac`?
```suggestion
  auto cc = static_cast<nntrainer::ClContext *>(eg.getRegisteredContext("gpu"));
```

Reply:
Makes sense!